### DISCUSSION TITLE
Ory Kratos + SvelteKit reference implementation

### DISCUSSION THREAD

**Discussion by DePasqualeOrg** [ory/.github#27] [Ideas]
SvelteKit is now in Release Candidate phase, with no more breaking changes expected until version 1.0. Perhaps this would be a good time to create a reference implementation for Ory Kratos using SvelteKit.

I found [this implementation](https://github.com/micleyman/sveltekit-ory-starter) using a now outdated version of SvelteKit, but SvelteKit has changed quite a bit since then, and I'm having trouble getting it to run. Perhaps the quickest path forward would be to [migrate](https://github.com/sveltejs/kit/discussions/5774) this to the latest version of SvelteKit.

**Comment by vinckr** [comment]
Hey @DePasqualeOrg, 
great idea 🙌 
There is another example here: https://github.com/drejohnson/sveltekit-kratos (and one just in Svelte here: https://github.com/emrahcom/kratos-selfservice-svelte-node) - they are probably also outdated. 

I think the best course of action would be to migrate [this example](https://github.com/micleyman/sveltekit-ory-starter) to the [ory/examples](https://github.com/ory/examples) repo and update it to the latest version. I won't have time to work on this before December probably, so happy for any contributions - will also give support if anyone is up for this.

**Comment by bessey** [comment]
Resurrecting an old thread but in case others were looking for an up to date example here's mine
https://github.com/bessey/ory-kratos-sveltekit-demo standing on the shoulders of @karlis-vagalis's https://github.com/karlis-vagalis/kratos-selfservice

It uses SvelteKit 2.1 / Ory Kratos 1.1.0

**Reply by vinckr** [reply]
thanks for sharing! will add it to the [community resources](https://www.ory.sh/docs/ecosystem/community)
